applications/portfolioDrafts/portfolio/createPortfolioStep.ts

```diff
@@ -20,9 +21,28 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
     );
   }
 
-  const portfolioId = event.pathParameters?.portfolioDraftId;
-
+  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
+  if (!portfolioDraftId) {
+    return new ErrorResponse(
+      { code: ErrorCodes.INVALID_INPUT, message: "PortfolioDraftId must be specified in the URL path" },
+      ErrorStatusCode.BAD_REQUEST
+    );
```

Comment:
I'd prefer that we treat this like a regular 404. The user didn't give us bad data, they navigated to the wrong URL. This response should be treated the same as if we had queried the database and there wasn't any such portfolioDraft.

The short-circuiting is kinda nice though (even if we probably don't actually need it since the later ConditionExpression should always fail for an empty input).